.github/workflows/tests-smoke.yaml

```diff
@@ -54,15 +54,7 @@ jobs:
           persist-credentials: false
 
       - name: Check pre-flight clusterrole
-        run: |
-          cd install/kubernetes/cilium/templates
-          echo "Checking for differences between preflight and agent clusterrole"
-          diff \
-             -I '^[ ]\{2\}name: cilium.*' \
-             -I '^Keep file in sync with.*' \
-             -I '{{- if.*' \
-             cilium-agent/clusterrole.yaml \
-             cilium-preflight/clusterrole.yaml
+        run: ./contrib/scripts/check-preflight-clusterrole.sh
```

Comment:
what do think about calling this in Makefile target, also then print the suggestion like below (i.e. user didn't even need to find which script to run, just check github action log)?

https://github.com/cilium/cilium/blob/master/contrib/scripts/check-k8s-code-gen.sh#L22

Reply:
Not a bad idea, what would the make target be tho? 
`make k8s-clusterrole-check` ?

Reply:
it's a hardest question in our profession i.e. naming.

How about check-k8s-clusterrole for consistent to `generate-k8s-api`